urbanairship-react-native/package.json

```diff
@@ -15,29 +14,57 @@
   ],
   "devDependencies": {
     "@babel/core": "7.5.4",
-    "@babel/plugin-transform-flow-strip-types": "^7.4.4",
-    "@babel/runtime": "7.5.4",
+    "@react-native-community/bob": "^0.15.1",
     "@react-native-community/eslint-config": "0.0.5",
-    "@typescript-eslint/parser": "^1.13.0",
+    "@types/jest": "^24.0.15",
+    "@types/prop-types": "^15.7.3",
+    "@types/react": "^16.8.23",
+    "@types/react-native": "^0.60.1",
+    "@types/react-test-renderer": "^16.8.2",
     "acorn": "^7.1.1",
     "acorn-jsx": "^5.0.2",
     "documentation": "12.1.1",
     "eslint": "^5.16.0",
     "jest": "^24.9.0",
     "jetifier": "^1.6.4",
-    "metro-react-native-babel-preset": "0.55.0",
     "react": "16.10.1",
     "react-native": "0.61.1",
-    "typescript": "^3.6.3"
+    "ts-jest": "^24.0.2",
+    "typedoc": "^0.17.7",
+    "typescript": "^3.9.5"
   },
-  "peerDependencies" : {
-    "react": "^16.8",
-    "react-native": ">=0.60"
+  "peerDependencies": {
+    "react": "^16.10",
+    "react-native": ">=0.61.1"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/urbanairship/react-native-module.git"
   },
   "license": "Apache-2.0",
-  "typings": "./js/index.d.ts"
+  "typings": "./js/index.d.ts",
+  "@react-native-community/bob": {
+    "source": "js",
+    "output": "lib",
+    "targets": [
+      "commonjs",
+      "typescript"
+    ]
+  },
+  "jest": {
+    "preset": "ts-jest",
+    "modulePathIgnorePatterns": ["lib/"]
+  },
+  "scripts": {
+    "prepare": "bob build",
+    "test": "jest --verbose",
+    "generate-docs": "typedoc --out docs --exclude js/__tests__ js/"
+  },
+  "types": "lib/typescript/index.d.ts",
```

Comment:
index.d.ts is now autogenerated, so in theory people this should help keep it from getting out of sync.

Reply:
Looks like we are not storing it after it is generated. What happens in a fresh checkout?

Reply:
`lib/` is .gitignore, so the idea is that the directory and its contents will be auto-generated locally instead of being checked in to version control. Think of `lib/` more like an obj or bin folder and not source code.